cypress/e2e/individual_therapist_spec.cy.js

```diff
@@ -0,0 +1,53 @@
+describe('Individual Therapist Flows', () => {
+
+  beforeEach(() => {
+    cy.visit('http://localhost:3000/1')
+  });
+
+  it('Should display the therapist\'s information', () => {
+    cy.get('.individual-section')
+      .should('exist')
+    cy.get('img')
+      .should('exist')
+      // .contains('https://loremflickr.com/300/300')
+    cy.get('h2')
+      .should('exist')
+      .contains('Sarah Jones')
+    cy.get('.individual-section > :nth-child(3)')
```

Comment:
We should probably add some data-cy tags so we can target each element with more accuracy 

[Cypress Docs](https://docs.cypress.io/api/cypress-api/selector-playground-api#Arguments)